test/functionalTest/cases/2101_Subscription_is_not_created_with_attribute_names_equals/Subscription_is_not_created_with_attribute_names_equals.test

```diff
@@ -0,0 +1,409 @@
+01. POST /v2/subscriptions, to create subscription with N attributes with the same name in attrs and see error
+==============================================================================================================
+HTTP/1.1 400 Bad Request
+Content-Length: 73
+Content-Type: application/json
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+Date: REGEX(.*)
+
+{
+    "description": "duplicated attribute /temperature/",
+    "error": "BadRequest"
+}
+
+
+02. POST /v2/subscriptions, to create subscription with N attributes with the same name in condition and see error
+==================================================================================================================
+HTTP/1.1 400 Bad Request
+Content-Length: 70
+Content-Type: application/json
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+Date: REGEX(.*)
+
+{
+    "description": "duplicated attribute /pressure/",
+    "error": "BadRequest"
+}
+
+
+03. POST /v2/subscriptions, to create subscription with N attributes with the same name in exceptAttrs and see error
+====================================================================================================================
+HTTP/1.1 400 Bad Request
+Content-Length: 73
+Content-Type: application/json
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+Date: REGEX(.*)
+
+{
+    "description": "duplicated attribute /temperature/",
+    "error": "BadRequest"
+}
+
+
+04. POST /v2/subscriptions, to create subscription with N attributes with different name in attrs and see no error
+==================================================================================================================
+HTTP/1.1 201 Created
+Content-Length: 0
+Location: /v2/subscriptions/REGEX([0-9a-f]{24})
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+Date: REGEX(.*)
+
+
+
+05. POST /v2/subscriptions, to create subscription with N attributes with different name in condition and see no error
+======================================================================================================================
+HTTP/1.1 201 Created
+Content-Length: 0
+Location: /v2/subscriptions/REGEX([0-9a-f]{24})
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+Date: REGEX(.*)
+
+
+
+06. POST /v2/subscriptions, to create subscription with N attributes with different name in exceptAttrs and no error
+====================================================================================================================
+HTTP/1.1 201 Created
+Content-Length: 0
+Location: /v2/subscriptions/REGEX([0-9a-f]{24})
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+Date: REGEX(.*)
+
+
+
+07. POST /v2/subscriptions, to create subscription with N attributes with the same name in metadata and see error
+=================================================================================================================
+HTTP/1.1 400 Bad Request
+Content-Length: 73
+Content-Type: application/json
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+Date: REGEX(.*)
+
+{
+    "description": "duplicated attribute /temperature/",
```

Comment:
In this case the description is a bit missleading, as the problem is not in attributes but in metadata. Maybe it should be changes to something more neutral, like:

```suggestion
    "description": "duplicated item in array /temperature/",
```

Reply:
Fixed in https://github.com/telefonicaid/fiware-orion/pull/4025/commits/81e67f5fb86e4e61d4879e6ee8924b88fdd5d5cb